tests/unit/test-extension-runners/test.extension-runners.js

```diff
@@ -368,6 +405,24 @@ describe('util/extension-runners', () => {
       );
     });
 
+    it('lints changed files', () => {
```

Comment:
@rpl i've just noticed that there are no tests for `MultiExtensionRunner` in test.run.js. before i continue, should l i write these? just one basic to test all passed param (including `desktopNotifications` which is [defined but never used](https://github.com/mozilla/web-ext/blob/master/tests/unit/test-cmd/test.run.js#L49)) would already be something